fastembed/text/onnx_text_model.py

```
@@ -68,8 +68,8 @@ def _load_onnx_model(
     def load_onnx_model(self) -> None:
         raise NotImplementedError("Subclasses must implement this method")
 
-    def tokenize(self, documents: list[str], **kwargs: Any) -> list[Encoding]:
-        return self.tokenizer.encode_batch(documents)  # type: ignore[union-attr]
+    def tokenize(self, texts: list[str], **kwargs: Any) -> list[Encoding]:
```

Ruff (0.14.1)

Unused method argument: `kwargs`

(ARG002)